### PR TITLE
Add rekt news

### PR DESCRIPTION
**Pull request by jonasvag**
## Adding a tech newsletter
- [x] Suggestion is not a duplicate
- [x] Suggestion is under the proper category
- [x] Name and link to the newsletter is included 
- [x] Name and link are spelled correctly

### CODE REVIEW

**Comment by jonasvag** [comment]
Hello, can this be approved?
https://rekt.news/

File: README.md [hunk 1]
```diff
@@ -163,5 +163,8 @@ We would love to know which tech newsletters you think deserve to be included in
     > Join 36,127 people & get a hand picked list of the best user experience design links every week. Curated by Kenny Chen & published every Monday.
 
 ## BLOCKCHAIN
+
 - [IBM Supply Chain & Blockchain Blog](https://www.ibm.com/blogs/blockchain/category/blockchain-newsletter/)
     > Monthly topics garnering news coverage and attention within the blockchain industry.
+- [Rekt News](https://doshbot.us8.list-manage.com/subscribe?u=4db8948c1d40f1c0e7af494fd&id=798585766e/)
```
Comment (jdwilkin4) [inline README.md:169]: The link you provided is not working.
Let's change it to this one here

```suggestion
- [Rekt News](https://rekt.news/)
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by jonasvag** [comment]
Updated by suggestion

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**